test/integration/test_js_errors.rb

```diff
@@ -0,0 +1,46 @@
+require File.expand_path(File.join(File.dirname(__FILE__), '..', 'helper'))
+require 'selenium-webdriver'
+require 'capybara'
+require 'capybara/dsl'
+
+def console_logs(page, level = :severe, &block)
+  yield page.driver.browser.manage.logs.get(:browser).select{|log| log.level == level.to_s.upcase }
+end
+
+def expected_errors
+  [
+    %r{Refused to apply style from 'http:.*/gollum/create/custom.css'}
+  ]
+end
+
+context 'Frontend with mathjax' do
+  include Capybara::DSL
+  
+  setup do
+    @path = cloned_testpath("examples/lotr.git")
+    @wiki = Gollum::Wiki.new(@path)
+    Precious::App.set(:gollum_path, @path)
+    Precious::App.set(:wiki_options, {mathjax: true})
+    Capybara.app = Precious::App
+    Capybara.default_driver = :selenium_chrome
+    Capybara.server = :webrick
+  end
+  
+  test 'expected asset errors' do
+    test_routes = ['/', '/create/Foobar.rst']
+    test_routes.each do |route|
```

Comment:
If this test breaks on one of the two routes, it will take more time to debug why this test is broken.

Perhaps we should add a distinct test block for each route that we care about?